storage/tests/unit/test_blob.py

```diff
@@ -1364,8 +1364,8 @@ def _do_upload_helper(
                 client, stream, content_type, size, num_retries,
                 predefined_acl)
 
-    def test__do_upload_without_chunk_size(self):
-        self._do_upload_helper()
+    def test__do_upload_small_file(self):
+        self._do_upload_helper(size=100)
 
     def test__do_upload_with_chunk_size(self):
         chunk_size = 1024 * 1024 * 1024  # 1GB
```

Comment:
Should this be 1GB or `_DEFAULT_CHUNKSIZE`?

Reply:
This could be changed to this for sure. It just needs to be smaller then 5MB so it uses the non-resumable codepath.